build.gradle.kts

```diff
@@ -15,8 +15,13 @@ repositories {
         url = uri("https://jitpack.io")
     }
 }
+val hivemq: Configuration by configurations.creating { isCanBeConsumed = false; isCanBeResolved = false }
+
+val operator: Configuration by configurations.creating { isCanBeConsumed = false; isCanBeResolved = false }
 
 dependencies {
+    hivemq("com.hivemq:hivemq")
+    operator("com.hivemq:hivemq-operator")
```

Comment:
Can't this be achieved with a https://docs.gradle.org/current/userguide/composite_builds.html ?

Reply:
@SgtSilvio can you confirm if this is correct please?

Reply:
These are just dependencies, and actually included builds are used.